composer.json

```diff
@@ -13,7 +13,7 @@
         }
     ],
     "require": {
-        "php": "^7.2",
+        "php": "^7.2||^8.0",
```

Comment:
```suggestion
        "php": ">=7.2",
```
:question:

Reply:
Good question. I know there was a debate about that on Symfony. Dunno the pros and cons.

Reply:
Do you want to support 9.0 in this release? That's why `"^7.2||^8.0"` is better.